### PR TITLE
[WebProfilerBundle] Fixes weird spacing in log message context/trace output

### PR DESCRIPTION
**Pull request by jennevdmeer**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        | 

This is a fix for a small issue created in bcb3329279a0774861e0c37155867ff45f7b7b6f. Where there is space created in the expanded log message context/trace output as seen below while preserving the space between the log message level, and context and trace toggles.

image

### CODE REVIEW

**Review by stof — APPROVED** [review]

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig [hunk 1]
```diff
@@ -1163,7 +1163,7 @@ table.logs .log-metadata {
 table.logs .log-metadata span {
```
Comment (stof) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig]: this might need to use `> span` too, to avoid affecting spans in the nested dump

Comment (jennevdmeer) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig]: Yes indeed, I didn't see any changing behavior before so I left it. But it causes some weird wrapping
image
the `$exists...` should start on the same line as the `>`. 

tldr I've added it.

**Review by chalasr — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @jennevdmeer.
